lib/librato/collector/aggregator.rb

```diff
@@ -5,14 +5,17 @@ class Collector
     # maintains storage of timing and measurement type measurements
     #
     class Aggregator
-      SOURCE_SEPARATOR = '$$'
+      SEPARATOR = "$$"
 
       extend Forwardable
 
-      def_delegators :@cache, :empty?, :prefix, :prefix=
+      def_delegators :@cache, :add_tags, :empty?, :prefix, :prefix=
```

Comment:
Since the tracker usually uses a single aggregator instance does it make sense to allow changes to the tagset after creation here?